scripts/replicate.py

```diff
@@ -0,0 +1,62 @@
+import timeit
+import argparse
+import json
+
+from aws_replicate import AWSBucketReplication
+from deletion import delete_objects_from_cloud_resources
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(title="action", dest="action")
+
+    aws_replicate_cmd = subparsers.add_parser("aws_replicate")
+    aws_replicate_cmd.add_argument("--global_config", required=True)
+    aws_replicate_cmd.add_argument("--bucket", required=True)
+    aws_replicate_cmd.add_argument("--manifest_file", required=True)
+    aws_replicate_cmd.add_argument("--thread_num", required=True)
+
+    aws_indexing_cmd = subparsers.add_parser("indexing")
+    aws_indexing_cmd.add_argument("--global_config", required=True)
+    aws_indexing_cmd.add_argument("--manifest_file", required=True)
+    aws_indexing_cmd.add_argument("--thread_num", required=True)
+
+    redact_cmd = subparsers.add_parser("readact")
+    redact_cmd.add_argument("--manifest_file", required=True)
+    redact_cmd.add_argument("--log_file", required=True)
+
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    start = timeit.default_timer()
+
+    args = parse_arguments()
+    if args.action == "aws_replicate":
```

Comment:
is this only used for aws replicating? sorta confusing that there's a `aws_replicate.py` file and a `replicate.py` file that takes an aws arg. would be nice to consolidate. I could be misunderstanding the different use-cases though

Reply:
Agree. It is sorta messy. Originally we want to support both google and aws replicate with multiple thread then we switch to google data flow. The repos should be divided into 2 repos, one for aws and one for google. I will spend time for refactor code in the next release